tests/packages/run_all.py

```diff
@@ -19,51 +19,48 @@
 #     limitations under the License.
 #
 
-import os, sys
+import os
+import sys
 
-# Find nuitka package relative to us.
-sys.path.insert(
-    0,
-    os.path.normpath(
-        os.path.join(
-            os.path.dirname(os.path.abspath(__file__)),
-            "..",
-            ".."
-        )
-    )
-)
 from nuitka.tools.testing.Common import (
+    compareWithCPython,
+    createSearchMode,
+    getTempDir,
     my_print,
     setup,
-    createSearchMode,
-    compareWithCPython,
     withExtendedExtraOptions,
-    getTempDir
 )
+from nuitka.tools.testing.SearchModes import SearchModeAll
+
+# Find nuitka package relative to us.
+sys.path.insert(
+    0,
+    os.path.normpath(
+        os.path.join(os.path.dirname(os.path.abspath(__file__)), "..", "..")
+    ),
+)
+
```

Comment:
I noticed you did add the isort:start correctly for one runner, but basically all that to that sys.path insertion need to do it, sorry, that is a bunch.